website/content/docs/k8s/installation/install-cli.mdx

```diff
@@ -247,6 +247,25 @@ You can include the `-auto-approve` option set to `true` to proceed with the ins
 
 The pre-install checks may fail if existing `PersistentVolumeClaims` (PVC) are detected. Refer to the [uninstall instructions](/consul/docs/k8s/operations/uninstall#uninstall-consul) for information about removing PVCs.
 
+## Custom installation
+
+You can create a values file and specify parameters to overwrite the default Helm chart installation. Add the `-f` and specify your values file to implement your configuration, for example:
+
+```shell-session
+$ consul-k8s install -f consulValues.yaml
+``` 
+
+### Install Consul on OpenShift clusters
+
+Red Hat OpenShift is a security-conscious, opinionated wrapper for Kubernetes. To install Consul on OpenShift-managed Kubernetes, add the following parameters to the `global` field of your [custom installation](#custom-installation) values file:
+
+```yaml
+  openshift:
+    enabled: true
+ ```
```

Comment:
I think it's clearer for people who are scanning to include the `global` as the root. They may not see the indentation immediately and it can mess them up if they are copy/pasting. 

```suggestion
[Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) is a security-conscious, opinionated wrapper for Kubernetes. To install Consul on OpenShift-managed Kubernetes, set `global.openshift.enabled=true` in your [custom installation](#custom-installation) values file:

```yaml
global:
  openshift:
    enabled: true
```
```